webpack.config.js

```diff
@@ -113,13 +111,12 @@ const config = {
 		new MiniCssExtractPlugin({
 			filename: `[name]${prefix}.css`,
 		}),
-		new CopyWebpackPlugin([{
-			from: './assets/images/',
-			to: 'images',
-			ignore: [
-				'.DS_Store'
-			]
-		}]),
+		new CopyWebpackPlugin({
+			patterns: [{
+                        	from: './assets/images/',
+                        	to: 'images',
```

Comment:
include the .DS_Store ignore (unless that's default now).

Reply:
I'll look into this in the next few days. they removed the top level ignore but there's some replacement.